.travis.yml

```diff
@@ -13,9 +13,6 @@ git:
   depth: 99999999
 before_install:
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-script:
-  - julia -e 'Pkg.clone(pwd());
-              Pkg.test("Glob", coverage=true)'
 after_success:
   - julia -e 'cd(Pkg.dir("Glob"));
```

Comment:
Seems to be failing? I'm not sure how to do this now, given that `Pkg.dir` is deprecated.

Reply:
> ┌ Warning: `Pkg.dir(pkgname, paths...)` is deprecated; instead, do `import Glob; joinpath(dirname(pathof(Glob)), "..", paths...)`.
> └ @ Pkg.API /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v0.7/Pkg/src/API.jl:454

Reply:
But that's terrible. So I also opened https://github.com/travis-ci/travis-ci/issues/9982 to make this the default behavior.

Reply:
Also probably wrong, since we don't want to do `import Glob` in this process. @StefanKarpinski how do we correctly get the path to something without loading it with the new `loading.jl` code?

Reply:
Oh, actually, I guess the issue is that the old Pkg code needed to make a copy of this code. In the new Pkg manager, this line should be unnecessary?